public/partials/farmer_profile_large.html

```diff
@@ -98,119 +98,130 @@ <H1 class="farmerSummary">{{farmer.fa_first_name}} {{farmer.fa_middle_name}} {{f
             </md-card>
 
             <md-card>
-                <md-card-content class="{{newFarm ? 'dark':'white'}}">
```

Comment:
@nick22891 notice that i manipulate the presentation by doing switching classes based on a condition.  This is important and makes developing the application with team members easier and more efficient.